flight/PiOS/STM32/pios_spi.c

```diff
@@ -89,8 +89,8 @@ void dma_init(DMA_Stream_TypeDef *stream, SPI_TypeDef *spi, uint32_t channel, bo
 
 void PIOS_SPI_InitializeDMA(struct pios_spi_dev *dev)
 {
-	dma_init(dev->cfg->dma_send_stream, dev->cfg->regs, dev->cfg->dma_send_channel, TRUE);
-	dma_init(dev->cfg->dma_receive_stream, dev->cfg->regs, dev->cfg->dma_receive_channel, FALSE);
+	dma_init(dev->cfg->dma_send_stream, dev->cfg->regs, dev->cfg->dma_send_channel, true);
+	dma_init(dev->cfg->dma_receive_stream, dev->cfg->regs, dev->cfg->dma_receive_channel, false);
```

Comment:
Should really fix the original commit so the change is atomic, otherwise bisection is sad. :/